### PR TITLE
METAL-2772/operator sdk v0.18.0

### PR DESCRIPTION
**Pull request by hyunysmile**
operator sdk v0.13.0 -> v0.18.0
followed guide line: https://sdk.operatorframework.io/docs/upgrading-sdk-version/v0.18.0/

### CODE REVIEW

File: helm/db-operator/templates/rbac.yaml [hunk 1]
```diff
@@ -25,27 +28,34 @@ rules:
   - services
```
Comment (dabde) [inline helm/db-operator/templates/rbac.yaml:28]: is database here needed? should already be covered by the api rule (kci.rocks), and think db operator is not create any database. Only verbs in question would be "patch".

**Review by dabde — DISMISSED** [review]
only minor comments.

File: helm/db-operator/templates/rbac.yaml (hunk 1 above)
Comment (hyunysmile) [reply helm/db-operator/templates/rbac.yaml:28]: patch is used to update service see: https://github.com/kloeckner-i/db-operator/blob/master/pkg/controller/database/reconcileProxy.go#L89

Comment (hyunysmile) [reply helm/db-operator/templates/rbac.yaml:27]: updated

**Review by dabde — APPROVED** [review]

**Pull request merged**